w3af/core/ui/api/main.py

```diff
@@ -51,17 +52,47 @@ def parse_arguments():
     Parses the command line arguments
     :return: The parse result from argparse
     """
-    parser = argparse.ArgumentParser(description='REST API for w3af')
+    parser = argparse.ArgumentParser(description='REST API for w3af',
+                                     formatter_class=argparse.RawTextHelpFormatter)
 
     parser.add_argument('host:port', action='store',
                         help='Specify address where the REST API will listen'
                              ' for HTTP requests. If not specified 127.0.0.1:'
                              '5000 will be used.',
-                        default='127.0.0.1:5000',
-                        nargs='?',
-                        type=parse_host_port)
+                        default=False,
+                        nargs='?')
+
+    parser.add_argument('-c',
```

Comment:
I believe that argparse provides a "type" for input files, if you specify that argparse will make sure the file exists, is readable, etc.